android/source/VideoLocker/src/org/edx/mobile/exception/LoginException.java

```diff
@@ -0,0 +1,23 @@
+package org.edx.mobile.exception;
+
+/**
+ * Created by rohan on 1/12/15.
+ */
+public class LoginException extends Exception {
+
+    private String messageLine1;
```

Comment:
Let's break this apart into two classes:

LoginErrorMessage, which just stores the two fields and LoginException which is just an Exception subclass with a LoginErrorMessage member.

This way we can pass error messages around without having to actually generate an exception.